src/store/modules/game.js

```diff
@@ -168,6 +169,8 @@ export default {
       if (Object.hasOwnProperty.call(newGame, 'p1Ready')) state.p1Ready = newGame.p1Ready;
       if (Object.hasOwnProperty.call(newGame, 'passes')) state.passes = newGame.passes;
       if (Object.hasOwnProperty.call(newGame, 'players')) state.players = cloneDeep(newGame.players);
+      if (Object.hasOwnProperty.call(newGame, 'spectatingUsers'))
+        state.spectatingUsers = newGame.spectatingUsers;
```

Comment:
Better to use braces for multi-line
```suggestion
      if (Object.hasOwnProperty.call(newGame, 'spectatingUsers')) {
        state.spectatingUsers = newGame.spectatingUsers;
      }
```